spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -168,12 +168,14 @@ def setup_toolbar(self):
 
         goback_btn = create_toolbutton(
                 self, icon=ima.icon('ArrowBack'),
-                tip=_("Previous Figure"),
+                tip=_("Previous Figure (%s)" %
```

Comment:
Please use the new [format](https://docs.python.org/3.4/library/functions.html#format) method for str. I think this is what we try to use now whenever we can. So what I mean is to do this instead:
```python
tip=_("Previous Figure ({})".format(
    get_shortcut('plots', 'previous figure')))
```